src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -200,7 +200,7 @@ private User prepareShadowUser(UUID userId, UserTenantEntity userTenantEntity, F
       User userOptional = getUser(userId);
 
       if (Objects.nonNull(userOptional.getId())) {
-        user.setId(UUID.randomUUID().toString());
+        user.setId(userId.toString());
```

Comment:
made shadow userId the same as real userId to simplify many things and according this doc:
https://wiki.folio.org/display/DD/Shadow+Users 

"mod-consortia will create a user(Shadow User) in the Target Tenant with the same user identifier as the user in the home tenant. The Shadow User will be created with permissions of the user that is trying to create the affiliation."